src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/overview/overview.component.ts

```diff
@@ -29,14 +36,25 @@ export class OverviewComponent implements OnInit, OnDestroy {
   peersExist = true;
   siteName: any;
   status: ViewCacheStatus;
+  rbdPoolExists: boolean;
+  rbdServiceExists$: Observable<boolean>;
+  orchAvailable$: Observable<boolean>;
+  rbdServiceExists: boolean;
+
+  icons = Icons;
   private subs = new Subscription();
 
   constructor(
     private authStorageService: AuthStorageService,
     private rbdMirroringService: RbdMirroringService,
-    private modalService: ModalService
+    private modalService: ModalService,
+    private cephServiceService: CephServiceService,
+    private poolService: PoolService,
+    private taskWrapper: TaskWrapperService,
+    private orchService: OrchestratorService
   ) {
     this.permission = this.authStorageService.getPermissions().rbdMirroring;
+    this.permission = this.authStorageService.getPermissions().pool;
```

Comment:
Aren't we overwriting the previous value? tslint should have caught this...

BTW, @nSedrickm created [a new directive](https://github.com/ceph/ceph/pull/41034/files) in order to tag visibility of HTML components to permissions and remove the boilerplate code of injecting the `AutStorageService` and so on...